neo/Ledger/Blockchain.cs

```diff
@@ -583,7 +583,18 @@ private void Persist(Block block)
                             using (ApplicationEngine engine = new ApplicationEngine(TriggerType.Application, tx_invocation, snapshot.Clone(), tx_invocation.Gas))
                             {
                                 engine.LoadScript(tx_invocation.Script);
-                                engine.Execute();
+                                foreach (IDebuggerPlugin plugin in Plugin.DebuggerPlugins)
+                                {
+                                    if (plugin.DebuggerActive)
```

Comment:
This verification could be inside the plugin for leaving this Core file cleaner.

Reply:
Basically here I am trying to make sure two plugins don't try and execute the same engine twice - if you leave it up to the plugins to enforce that it could be messy